### PR TITLE
Mt 112

### PR DESCRIPTION
**Pull request by yuri-vayne-jala**
I solved conflict with the last commit

### CODE REVIEW

File: IDT.MoneyTransfer/Main/Utilities/Constants.cs [hunk 1]
```diff
@@ -9,5 +9,6 @@ public class Constants
         public const string TxtSearchState = "input[class='select2-search__field']";
         public const string PhoneNumberExistentSender = "5555555555";
         public const string LastNameExistentSender = "Vayne";
+        public const string DBConnection = "user id=psf;password=retailqa_psf;data source=" + "(DESCRIPTION=(ADDRESS=(PROTOCOL=tcp)" + "(HOST=ex02qa03-vip.corp.idt.net)(PORT=1522))(CONNECT_DATA=" + "(SERVICE_NAME=bossqa.db.idt.net)))";
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/Constants.cs]: This constant should read values of App.config

File: IDT.MoneyTransfer/Main/Utilities/DB/DBConnection.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿namespace IDT.MoneyTransfer.Main.Utilities.DB
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/DB/DBConnection.cs]: This class should be moved to Core and it should be a Header comments

File: IDT.MoneyTransfer/Main/Utilities/DB/DBConnection.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        /// <summary>
+        /// Gets values of db using an specific query.
+        /// </summary>
+        /// <param name="query">the query to get values of db</param>
+        /// <returns>the table with values related to query</returns>
+        public static DataTable GetData(string query)
+        {
+            DataTable table = new DataTable();
+            OracleConnection connection = GetConnection(Constants.DBConnection);
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/DB/DBConnection.cs]: Check if StyleCop suggest that you should use **this.**GetConnection

File: IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿namespace IDT.MoneyTransfer.Main.Utilities.DB
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs]: I think that it should be moved to Core

File: IDT.MoneyTransfer/Tests/UI/Features/MoneyTransfer.feature [hunk 5]
```diff
@@ -52,11 +52,24 @@ Given I login to retailer portal
...
-#		And I select an existent Sender
-#		And I select an existent Recipient
+#		And I fill the new Sender form with following data:
+#			| PhoneNumber			|FirstName		|  LastName     | MaindenName	| Email							| Addres1	|Addres2	|PostalCode     |City					|State		|
+#			|8888888888				| auto_sender04 | vayne         | rivera		|auto_sender03@mailinator.com	|av test	|av test	|75202			| JACKSONVILLE, US-AR   |ARKANSAS	|
+#		And I fill the new Recipient form with following data:
+#			| PhoneNumber			|FirstName		|  LastName     | MaindenName	| Email							| Addres1	|Addres2	|PostalCode     |City						|State					|
+#			|2222222222				| recipient05	| vayne         | rivera		|auto_recipient05@mailinator.com|av test	|av test	|78975			| DISTRITO FEDERAL, MX-BCN  |BAJA CALIFORNIA NORTE	|
 #		And I send "10" USD with "MXN" Receiving Currency
 #		And I select "Cash Pickup" on Delivery Method
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Tests/UI/Features/MoneyTransfer.feature]: For this step we need to know the location where the Cash will be delivered. Coordinate with Reinaldo for this step

File: IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs [hunk 6]
```diff
@@ -1,4 +1,6 @@
-﻿namespace IDT.MoneyTransfer.Tests.UI.StepsDefinitions
+﻿#pragma warning disable SA1101
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs]: Why we had this lines ? #pragma?

File: IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs [hunk 7]
```diff
@@ -104,6 +106,18 @@ public void WhenISelectForPrintingOptions(string printingOption)
...
+        public void WhenICloseTheSuccessTransactionNotification()
+        {
+            receiptPage.IsDisplayed();
+            receiptPage.GetNotificationMessage();
+        }
+
+        [Then(@"The wwCreateRecipient request should return a response indicating ""(.*)""\.")]
+        public void ThenTheWwCreateRecipientRequestShouldReturnAResponseIndicating_(string p0)
+        {
+            Assert.IsTrue(true);
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs]: It does not looks a good Assert, we should perform a valid validation

File: IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs [hunk 8]
```diff
@@ -104,6 +106,18 @@ public void WhenISelectForPrintingOptions(string printingOption)
...
 
+        [When(@"I close the Success Transaction Notification")]
+        public void WhenICloseTheSuccessTransactionNotification()
+        {
+            receiptPage.IsDisplayed();
+            receiptPage.GetNotificationMessage();
+        }
+
+        [Then(@"The wwCreateRecipient request should return a response indicating ""(.*)""\.")]
+        public void ThenTheWwCreateRecipientRequestShouldReturnAResponseIndicating_(string p0)
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs]: the parameter is p0 ?

File: IDT.MoneyTransfer/Main/Utilities/Constants.cs (hunk 1 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/Utilities/Constants.cs]: I agree ,I will update it

File: IDT.MoneyTransfer/Main/Utilities/DB/DBConnection.cs (hunk 2 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/Utilities/DB/DBConnection.cs]: I agree ,I will move it

File: IDT.MoneyTransfer/Main/Utilities/DB/DBConnection.cs (hunk 3 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/Utilities/DB/DBConnection.cs]: the use of this is only when the parameter and the class attribute have the same name,in that case is use it "this" e.g
class test{
   string name;
   public(string name)
   {
     this.name = name;
   }
}
we need to disable that rule of style cop.

File: IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs (hunk 4 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs]: okas

File: IDT.MoneyTransfer/Tests/UI/Features/MoneyTransfer.feature (hunk 5 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Tests/UI/Features/MoneyTransfer.feature]: ok, I will talk with him.

File: IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs (hunk 6 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs]: the pragma SA1101 disable the rule in style cop for "this".

File: IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs (hunk 8 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs]: I will update it

File: IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs (hunk 7 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs]: I put this way due  the db connection was not complete. now It is completed I will update this step.

File: IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs (hunk 4 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs]: Sorry , here I don't think this file should be in Core, due the query's will be different for each framework. e.g.
The querys on Pinless, will not be useful for MT.

Comment (wMercadoJala) [reply IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs]: ok np

Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs]: Add Class header

File: IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+{
+    class Querys
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs]: Add summary for the class

File: IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs [hunk 10]
```diff
@@ -1,4 +1,6 @@
-﻿namespace IDT.MoneyTransfer.Tests.UI.StepsDefinitions
+﻿#pragma warning disable SA1101
+#pragma warning disable SA1124
+namespace IDT.MoneyTransfer.Tests.UI.StepsDefinitions
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs]: Add class headers

**Comment by yuri-vayne-jala** [comment]
Hey Walter , I completed the MT-112 and solve conflicts

File: IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs (hunk 9 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/Utilities/DB/Querys.cs]: done

File: IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs (hunk 10 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Tests/UI/StepsDefinitions/MoneyTransferSteps.cs]: done

File: IDT.MoneyTransfer/packages.config [hunk 11]
```diff
@@ -2,7 +2,11 @@
 <packages>
   <package id="Appium.WebDriver" version="3.0.0.2" targetFramework="net45" />
   <package id="Castle.Core" version="4.2.0" targetFramework="net45" />
+<<<<<<< HEAD
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/packages.config]: You should fix this merge conflic

**Pull request merged**